lucid/launcher.py

```diff
@@ -140,6 +140,9 @@ def launch(beamline, *, toolbar=None, row_group_key="location_group",
     # Re-enable sigint (usually blocked by pyqt)
     signal.signal(signal.SIGINT, signal.SIG_DFL)
 
+    # Silence the logger from pyPDB.dbd.yacc
+    logging.getLogger("pyPDB.dbd.yacc").setLevel(logging.WARNING)
```

Comment:
Ugh...

Reply:
Yeah... it sucks but it is all I can do right now at least. We can remove this line once it get addressed at `pyPDB`.